src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/MutationScripts/ClearCollect.txt

```diff
@@ -86,7 +86,19 @@ Errors: Error 22-24: The specified column 'a' does not exist.|Error 0-12: The fu
 Errors: Error 13-20: The value passed to the 'ClearCollect' function cannot be changed.
 
 >> ClearCollect("", "")
-Errors: Error 13-15: Invalid argument type (Text). Expecting a Table value instead.|Error 17-19: Invalid argument type (Text). Expecting a Record value instead.|Error 17-19: Invalid argument type. Cannot use Text values in this context.|Error 0-12: The function 'ClearCollect' has some invalid arguments.
+Errors: Error 13-15: The function 'ClearCollect' has some invalid arguments.|Error 17-19: Invalid argument type. Cannot use Text values in this context.|Error 0-12: The function 'ClearCollect' has some invalid arguments.
 
 >> ClearCollect(If(false,Blank()),r1)
 Errors: Error 31-33: The specified column 'a' does not exist.|Error 0-12: The function 'ClearCollect' has some invalid arguments.
+
+>> Set(t2, [1,2,3])
+Table({Value:1},{Value:2},{Value:3})
+
+>> ClearCollect(t2, 99)
+{Value:99}
+
+>> 0;t2
```

Comment:
>0;t2 [](http://example.com/codeflow?start=3&length=4)

Curious: why `0;t2` and not just `t2`?

Reply:
This is to avoid getting a "duplicated test case" error. So every time we want to check the content of a variable, we call `0;t2`, `1;t2`, and so on.